pkg/container/container.go

```diff
@@ -69,3 +63,51 @@ func CalculateID(c *Container) *ID {
 
 	return id
 }
+
+func (c *Container) GetVersion() *pkg.Version {
```

Comment:
can we get rid of `Get` for `getter` methods (according to go-way)?

Reply:
https://golang.org/doc/effective_go.html#Getters

Reply:
Yes, it is suitable there, thank you.